<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZECRWindowInformation</class>
 <widget class="QWidget" name="ZECRWindowInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblInformationText">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Your application has crashed unexpectedly !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The problem may be due a bug in application or a problem in your computer. We are sorry for the inconvience. Please restart the application and do not forget to save often.&lt;/p&gt;&lt;p&gt;You can help us to find the root cause of this problem.&lt;/p&gt;&lt;p&gt;We have crated a report about this crash that you can send to us through internet. This report is going to help us what happened in our application that caused it to crash. Therefore, it will help us to solve your problem and improve our software in the process.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnViewReport">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>View Content of the Report</string>
     </property>
     <property name="description">
      <string>View what we have collected from your computer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnViewPrivacyPolicy">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>View Privacy Policy</string>
     </property>
     <property name="description">
      <string>View what we are going to do with the information that we have collected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnGotoSupportPortal">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Go to Support Portal</string>
     </property>
     <property name="description">
      <string>Visit our support portal for help or bug report</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDontSend">
       <property name="text">
        <string>Dont Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
